src/solvers/dgsem_p4est/dg_2d.jl

```diff
@@ -20,32 +20,68 @@ function create_cache(mesh::P4estMesh{2}, equations, mortar_l2::LobattoLegendreM
 end
 
 
+# TODO: p4est interface performance, move and generalzie this function for 3D
+@inline function index_to_start_step(index::Symbol, idx_one::Int, idx_end::Int)
```

Comment:
I'm sure that I will need to rewrite this when working on 3D. Since there is already a TODO note, I don't really feel like writing throw-away documentation right now...

Reply:
Sounds good to me!